articles/azure-functions/durable/durable-functions-concepts.md

```diff
@@ -34,6 +34,25 @@ In the function chaining pattern, a sequence of functions executes in a specific
 
 You can use Durable Functions to implement the function chaining pattern concisely as shown in the following example:
 
+#### Precompiled C#
```

Comment:
Nit: The note below the C# script example mentions `There are subtle differences between writing a precompiled durable function in C# and writing a precompiled durable function in the C# script that's shown in the example.`

Since both are now examples, can we say something like `There are subtle differences between writing a precompiled durable function in C# and writing a precompiled durable function in C# script.` since both are now shown as examples?